enferno/admin/models/DynamicField.py

```
@@ -0,0 +1,425 @@
+        return super().save()
+
+    def create_column(self):
+        """Create the actual column in the entity table"""
+        model_class = self.get_entity_model()
+        table_name = model_class.__tablename__
+
+        try:
+            # Direct mapping from field_type to SQL type
+            sql_type = DynamicField.TYPE_MAP.get(self.field_type)
+            if not sql_type:
+                raise ValueError(f"Invalid field type: {self.field_type}")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)